ui/DesignSystem/Component/Dropdown.svelte

```diff
@@ -25,11 +41,12 @@
 
     expanded = !expanded;
   };
+
   const hideMenu = () => {
     expanded = false;
   };
 
-  const optionSelectedHandler = event => {
+  const optionSelectedHandler = (event: CustomEvent<{ value: string }>) => {
```

Comment:
Yes - it's a `CustomEvent` dispatched by `Option.svelte`: https://github.com/radicle-dev/radicle-upstream/blob/master/ui/DesignSystem/Component/Dropdown/Option.svelte#L20

This could be exported from a lib file:
```ts
type OptionSelect = CustomEvent<{ value: string }>
```

But I was thinking that would be overkill, and might be better served by a general approach to typing dispatched events. What do you think?